k8sutils/k8sutils_test.go

```diff
@@ -27,7 +27,7 @@ import (
 func Test_CreateKubeClientSet(t *testing.T) {
 	var tempConfigFunc func() (*rest.Config, error)                               // must return getInClusterConfig to its original value
 	var tempClientsetFunc func(config *rest.Config) (kubernetes.Interface, error) // must return getK8sClientset to its original value
-	const kubeConfig = "/test/path/to/kubeconfig"
+	const kubeConfig = "$HOME/.kube/config"
```

Comment:
Why not just use KUBECONFIG. It's the standard variable that kubectl uses and can be passed into the test.